src/ReadCourseData.py

```diff
@@ -25,14 +26,17 @@ def read_course_proto(json_object, course_list, department_name):
                           be filled with course objects
       department_name: the department name, used to access the json file
                       and to record what courses are in this department
+      seat_dict: the dictionary containing act number and wl act number 
```

Comment:
nit: When adding comments for dictionary, a good way to help reader understand is to explain what is the key and what is the value for each entry. For example, 
```
Key: course numer(crn)
Value: A dictionary including enrollment information such as active cap(act), 
remaining(rem), waitlist remaining(wl_rem), and fetch time(fetch_time).
```